frontend/src/components/elements/ArrowDataFrame/styled-components.ts

```diff
@@ -27,38 +26,66 @@ export const StyledDataFrameContainer = styled.div<
   StyledDataFrameContainerProps
 >(({ width, theme }) => ({
   width,
-  border: `1px solid ${theme.colors.secondaryBg}`,
+  border: `1px solid ${theme.colors.fadedText10}`,
   boxSizing: "content-box",
 
   "& .table-top-right": {
-    overflowX: "hidden",
-    backgroundColor: theme.colors.secondaryBg,
+    overflow: "hidden !important",
   },
+
   "& .table-bottom-left": {
-    overflowY: "hidden",
-    backgroundColor: theme.colors.secondaryBg,
+    overflow: "hidden !important",
+    paddingBottom: "6px",
+  },
+
+  "& .table-bottom-right": {
+    overflow: "hidden !important",
```

Comment:
I think these overflows can be fixed with `enableFixedColumnScroll` and `enableFixedRowScroll`, right? [See Docs](https://github.com/bvaughn/react-virtualized/blob/master/docs/MultiGrid.md)

The name is weird, but the behavior for `[enableFixedRowScroll](https://github.com/bvaughn/react-virtualized/blob/v9.22.2/source/MultiGrid/MultiGrid.js#L516)` and `[enableFixedRowScroll](https://github.com/bvaughn/react-virtualized/blob/v9.22.2/source/MultiGrid/MultiGrid.js#L480)` is the same.

I'm not sure I understand what the hover is doing. I tried testing it out...but wasn't able to see the difference, but I might not be having a clear use case. In any case, this could be simplified.

Reply:
I tried it but then I can get this kind of misalignment on scroll:

<img width="157" alt="Screen Shot 2021-09-01 at 7 54 50 AM" src="https://user-images.githubusercontent.com/690814/131694459-83d4f468-ff03-4162-8d2c-a800d2db054e.png">

Reply:
Actually, I got a fix! Closing.